sdk/servicebus/Azure.Messaging.ServiceBus/samples/Sample03_SendReceiveSessions.md

```diff
@@ -34,10 +34,10 @@ Console.WriteLine(receivedMessage.SessionId);
 
 // we can also set arbitrary session state using this receiver
 // the state is specific to the session, and not any particular message
-await receiver.SetSessionStateAsync(Encoding.UTF8.GetBytes("some state"));
+await receiver.SetSessionStateAsync(new BinaryData(Encoding.UTF8.GetBytes("some state")));
```

Comment:
Should just pass the string directly.

Reply:
You can't do that. As I explained in the issue, there is no transitive property in C#:
![image](https://user-images.githubusercontent.com/17788297/94961288-36c81b00-04a9-11eb-9f0f-48475ed5c0dc.png)

Reply:
Fixed.